umbral/keys.py

```diff
@@ -117,6 +114,12 @@ def to_bytes(self, password: bytes=None, _scrypt_cost: int=20,
 
         return umbral_privkey
 
+    def __bytes__(self):
```

Comment:
We don't offer `__bytes__` on `UmbralPrivateKey` to prevent accidental misuse.

I think this should be removed.

Reply:
I agree with @tuxxy here.  What's the use case?

Reply:
None. It was a silly move :man_facepalming: ... Fixed.